serverless.ts

```diff
@@ -140,7 +144,10 @@ const serverlessConfiguration: AWS = {
     findSoftwareHandler,
     listSoftwareHandler,
     updateSoftwareHandler,
-    deleteSoftwareHandler
+    deleteSoftwareHandler,
+    listUsersHandler,
+    findUserHandler,
+    testFunctions
```

Comment:
Should test functions be a part of this PR? If you think they should then they should have a more descriptive name otherwise be removed